spec/models/draw_spec.rb

```diff
@@ -47,11 +47,12 @@
   describe '#group_sizes' do
     let(:draw) { FactoryGirl.build_stubbed(:draw) }
     it 'returns an array of group sizes in the draw' do
-      instance_spy('GroupSizesQuery', call: [1, 2]).tap do |q|
+      g = instance_spy('group_sizes_query', call: [1, 2]).tap do |q|
```

Comment:
I'm not sure that we can just test that the method was called on the query object here -- we be ensuring that the return value of `group_sizes` is correct, so we need to set a return value for `call`